code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -601,6 +698,16 @@
 			special_light_key = "miner"
 		if("Spider Miner")
 			cyborg_base_icon = "spidermin"
+		if("Marina")
+			cyborg_base_icon = "marinamin"
+		if("Heavy")
+			cyborg_base_icon = "heavymin"
+		if("Can")
+			cyborg_base_icon = "canmin"
+		if("Droid")
+			cyborg_base_icon = "cminer"
+		if("Sleek")
+			cyborg_base_icon = "sleekmin"

```

Comment:
comment these

Reply:
lower case, space after the //, then after the austation start, a -- then the reason for the edit
austation end just needs a space after the // and lowercase

Reply:
misspelled austation, add the `begin` after the `// austation`